api/parser_handler.go

```diff
@@ -32,15 +32,16 @@ func ParserHandler(w http.ResponseWriter, r *http.Request, mysqlStorage *storage
 		return
 	}
 
-	// 3. Store claim + claim result into the database
-	if !util.StoreClaimFileInDatabase(db, claimResults, partnerName, ocpVersion, executedBy) {
-		util.WriteError(w, util.ClaimFileError, err.Error())
+	// 3. Store file to S3
```

Comment:
So you store the file at first and then put into the database. But this order is problematic in case the rollback happens. The file would be stored inside S3. In case transaction rollback happens, can you delete this file inside S3?